### ISSUE TITLE
Does not build on Windows

### ISSUE THREAD

**Issue by cretz** [DataDog/temporalite#46]
## Expected Behavior

Builds on Windows.

## Actual Behavior

Not building. From a fresh recursive git clone, a `go build ./cmd/temporalite` gives:

    C:\Users\user\go\pkg\mod\github.com\temporalio\ui-server@v0.1.1-0.20211223210854-4484839e0398\server\server.go:47:113: pattern generated/ui/_app/assets/pages/namespaces/\[namespace\]/queues/\[queue\]/*: syntax error in pattern

Since the UI server was added, https://github.com/temporalio/ui-server/issues/104 is causing problems with Windows build. Supporting the UI as a build flag (or one prefixed with a bang for opt-out) per #33 could alleviate this temporarily. Also, I'd recommend adding Windows do the CI workflow.

This is on Go 1.17.7. I have not investigated the fix, this is just for tracking.

**Comment by feedmeapples** [comment]
For visibility https://github.com/temporalio/ui-server/issues/104#issuecomment-1051416368

**Comment by jlegrone** [comment]
This is blocked on the upstream ui-server issue.

In the mean time, Windows users can work around this by building Temporalite with the `headless` tag enabled:

```bash
go install -tags headless github.com/DataDog/temporalite/cmd/temporalite@latest
```

https://github.com/DataDog/temporalite/pull/47 added build/test support for Windows and macOS in addition to linux, so we should discover more platform specific issues before merging pull requests in the future.

**Comment by jlegrone** [comment]
This should be fixed as a result of the update to Go 1.18.
